RefactoringsExperiments/src/main/java/org/jetbrains/research/extractMethodExperiments/FalseRefactoringsExtractorCaller.java

```diff
@@ -0,0 +1,40 @@
+package org.jetbrains.research.extractMethodExperiments;
+
+import org.jetbrains.research.extractMethodExperiments.csv.models.Feature;
+import org.jetbrains.research.extractMethodExperiments.extractors.FalseRefactoringsExtractor;
+import org.jetbrains.research.extractMethodExperiments.utils.RepoListParser;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
+public class FalseRefactoringsExtractorCaller {
+    static void makeFileHeader(FileWriter fw) throws IOException {
+        for (int i = 0; i < 117; i++)
```

Comment:
117 is a "magic number",
i suggest introducing a field `private final int FEATURES_COUNT = 117` and use it here